src/schema/v2/reverseImageSearch.ts

```diff
@@ -69,6 +70,19 @@ export const ReverseImageSearchResult = new GraphQLObjectType({
       type: new GraphQLNonNull(ReverseImageSearchResultMatchRect),
       resolve: ({ query_match_rect }) => query_match_rect,
     },
+    artwork: {
+      type: ArtworkType,
+      resolve: ({ filepath }, _args, { artworkLoader }) => {
```

Comment:
`artworkLoader` should always exist so  can probably be used w/o checking.

Additionally, since the  user is providing a token this will use the  uncached `artworkLoader`. It's probably a tiny optimization to use the cached one (since it doesn't seem  like this field should try to return an unpublished artwork to its owner or that kind of thing).

To access that, destruct the last argument  to  `resolve` like so: `{ unauthenticatedLoaders: { artworksLoader } }`

Reply:
> Additionally, since the user is providing a token this will use the uncached artworkLoader. It's probably a tiny optimization to use the cached one (since it doesn't seem like this field should try to return an unpublished artwork to its owner or that kind of thing).

Good point. Thank you!